src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawPaneControllerToJob2dDriverAdapter.java

```diff
@@ -27,7 +28,10 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		setPosition(x,y);
+
+		DrawPanelController drawerController = DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.